lib/src/rewrite.rs

```diff
@@ -486,12 +487,7 @@ pub fn move_commits(
     options: &RebaseOptions,
 ) -> BackendResult<MoveCommitsStats> {
     if target_commits.is_empty() {
-        return Ok(MoveCommitsStats {
-            num_rebased_targets: 0,
-            num_rebased_descendants: 0,
-            num_skipped_rebases: 0,
-            num_abandoned: 0,
-        });
+        return Ok(Default::default());
```

Comment:
nit: even though this is used inconsistently throughout jj's code base, it is usually considered a better practice to write `MoveCommitsStats::default()` to make it clear what type is built here.

Reply:
Ah, thanks for the pointer. As you might be able to tell, I don't really write a lot of Rust, so I'm not familiar with quite a lot of these common practices.